### PR TITLE
Admin ID alias

### PR DESCRIPTION
**Pull request by tannermpeterson**

### CODE REVIEW

**Review by jperson — DISMISSED** [review]

File: frontend/pulse3d/src/components/account/AdminAccountOptions.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    });
+  };
+
+  const updateAccountSettingsEdits = (newValues) => {
+    setUpdateStatus(UPDATE_STATUSES.EDITING);
+
+    const { alias: newAlias } = newValues;
+    if (newAlias != null) {
+      let errorMsg = "";
+      if (newAlias.length > 0) {
```
Comment (luciipak) [inline frontend/pulse3d/src/components/account/AdminAccountOptions.js:94]: What happens if the user inputs something and then backspaces to length of 0?

File: frontend/pulse3d/src/components/account/AdminAccountOptions.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+          placeholder={accountSettings.alias ? "Leave empty to remove the current alias" : "None set"}
+          value={accountSettingsEdits.alias || ""}
+          tooltipText={
+            "Set an alias for the Customer ID field used when logging into a user account (6-128 characters)."
+          }
+          onChangeFn={(e) => {
+            updateAccountSettingsEdits({ alias: e.target.value });
+          }}
+        />
+        <ErrorText id={"Account Alias Error"} role="errorMsg">
```
Comment (luciipak) [inline frontend/pulse3d/src/components/account/AdminAccountOptions.js:153]: Are parentheses needed in the ID?

**Review by luciipak — APPROVED** [review]

File: frontend/pulse3d/src/components/account/AdminAccountOptions.js (hunk 1 above)
Comment (tannermpeterson) [reply frontend/pulse3d/src/components/account/AdminAccountOptions.js:94]: submitting with the input set to an empty string is fine, the BE will interpret that as clearing the ID

File: frontend/pulse3d/src/components/account/AdminAccountOptions.js (hunk 2 above)
Comment (tannermpeterson) [reply frontend/pulse3d/src/components/account/AdminAccountOptions.js:153]: What do you mean by that?

Comment (luciipak) [reply frontend/pulse3d/src/components/account/AdminAccountOptions.js:153]: can't it be id="Account Alias Error" or id="accountAliasError"?

Comment (tannermpeterson) [reply frontend/pulse3d/src/components/account/AdminAccountOptions.js:153]: yeah it probably could

**Pull request merged**